go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/IBM/platform-services-go-sdk
+module github.com/manukm-ibm/platform-services-go-sdk
```

Comment:
Please revert this change.  It looks like it's causing errors in the build.

Reply:
Done. This was a temporary change to test my TF scripts